multisig/simulations/zksyncsepolia/vip-007/index.ts

```diff
@@ -0,0 +1,131 @@
+import { expect } from "chai";
+import { Contract, Signer } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { initMainnetUser } from "src/utils";
+import { forking, pretendExecutingVip } from "src/vip-framework";
+import {
+  RewardsDistributorConfig,
+  checkRewardsDistributor,
+  checkRewardsDistributorPool,
+} from "src/vip-framework/checks/rewardsDistributor";
+
+import vip007, {
+  COMPTROLLER_CORE,
+  REWARD_DISTRIBUTOR_CORE_0,
+  REWARD_TOKEN_SPEED,
+  VUSDC_E_CORE,
+  VUSDT_CORE,
+  VWBTC_CORE,
+  VWETH_CORE,
+  VZK_CORE,
+  XVS,
+  XVS_REWARD_AMOUNT,
+  XVS_STORE,
+  XVS_VAULT_PROXY,
+} from "../../../proposals/zksyncsepolia/vip-007";
+import XVS_VAULT_ABI from "./abi/XVSVaultProxy.json";
+import XVS_ABI from "./abi/xvs.json";
+
+const { zksyncsepolia } = NETWORK_ADDRESSES;
+
+const BRIDGE = "0x760461ccB2508CAAa2ECe0c28af3a4707b853043";
+const TREASURY_AMOUNT = parseUnits("7200", 18);
+
+forking(3602113, async () => {
+  const provider = ethers.provider;
+  let bridgeSigner: Signer;
+  let xvsVault: Contract;
+  let xvs: Contract;
+
+  describe("Pre-VIP behaviour", async () => {
+    before(async () => {
+      bridgeSigner = await initMainnetUser(BRIDGE, parseUnits("1", 18));
+
+      xvsVault = new ethers.Contract(XVS_VAULT_PROXY, XVS_VAULT_ABI, provider);
+      xvs = new ethers.Contract(XVS, XVS_ABI, provider);
+
+      await xvs.connect(bridgeSigner).mint(zksyncsepolia.VTREASURY, TREASURY_AMOUNT, { maxFeePerGas: "20000000000" });
```

Comment:
I would add a VIP in this PR instead, bridging the needed XVS from BSC testnet to zkSync. So, first we'll execute the VIP bridging the XVS to the treasury on zkSync and them we will execute the TX's configuring the rewards

Reply:
[2793f09](https://github.com/VenusProtocol/vips/pull/348/commits/2793f09627521e9d595e592e60eae6ff73f9dfe5)